showroom/src/main/java/no/mhl/showroom/ui/views/InfiniteViewPager2.kt

```diff
@@ -15,6 +15,12 @@ class InfiniteViewPager2(
     attrs: AttributeSet?
 ) : FrameLayout(context, attrs) {
 
+    // region Constants
+    companion object {
+        const val EXCESS_ITEM_COUNT: Int = 2
```

Comment:
I guess this is as good a name as any 🙂  Still requires some thought to figure out _why_ we have excess items, so maybe a comment would help?